include/boost/geometry/extensions/contrib/ttmath_stub.hpp

```diff
@@ -103,6 +107,12 @@ namespace ttmath
         return two * ATan((sqrt(x * x + y * y) - x) / y);
     }
 
+    // needed in order to work with boost::geometry::math::fmod
+    template <uint Exponent, uint Mantissa>
+    inline Big<Exponent, Mantissa> fmod(Big<Exponent, Mantissa> const& x, Big<Exponent, Mantissa> const& y)
```

Comment:
After the changes done in the other PR here the name should be changed to `mod()` as well.